internal/controllers/user_devices_controller.go

```diff
@@ -1919,6 +1976,18 @@ type AdminRegisterUserDevice struct {
 	Verified    bool    `json:"verified"`
 }
 
+type GetUserDevicesByIntegration struct {
+	AuthorizationCode string `json:"authorizationCode"`
+	RedirectURI       string `json:"redirectUri"`
```

Comment:
```suggestion
	RedirectURI       string `json:"redirectURI"`
```

Reply:
`redirectURI` is more a Go covention as opposed to what a client like say JS uses

Reply:
I would do `redirectUri`.

Reply:
@KevinJoiner Does that make sense? I must admit I'm not super confident on this.